src/raftstore/store/peer.rs

```diff
@@ -990,11 +1057,15 @@ impl Peer {
         Ok(res)
     }
 
-    fn find_cb(&mut self, uuid: Uuid, term: u64, cmd: &RaftCmdRequest) -> Option<Callback> {
+    fn find_cb(&mut self,
+               uuid: Uuid,
+               term: u64,
+               cmd: &RaftCmdRequest)
+               -> Option<(Callback, Option<Timespec>)> {
```

Comment:
Not all pending commands `Store` passes to `Peer` has a `renew_lease_time`, it's convenient to make it an `Option`.